install/management/appliance.go

```diff
@@ -153,26 +126,27 @@ func (d *Dispatcher) getNetworkDevices(conf *configuration.Configuration) ([]typ
 	return devices, nil
 }
 
-func (d *Dispatcher) createApplianceSpec(conf *configuration.Configuration) (*types.VirtualMachineConfigSpec, error) {
+func (d *Dispatcher) createApplianceSpec(conf *metadata.VirtualContainerHostConfigSpec) (*types.VirtualMachineConfigSpec, error) {
 	netDevices, err := d.getNetworkDevices(conf)
 	if err != nil {
 		return nil, err
 	}
 
 	unitNumber = -1
+
 	spec := &types.VirtualMachineConfigSpec{
-		Name:     conf.DisplayName,
+		Name:     conf.Name,
 		GuestId:  "other3xLinux64Guest",
-		Files:    &types.VirtualMachineFileInfo{VmPathName: fmt.Sprintf("[%s]", conf.ImageDatastoreName)},
-		NumCPUs:  conf.NumCPUs,
-		MemoryMB: conf.MemoryMB,
+		Files:    &types.VirtualMachineFileInfo{VmPathName: fmt.Sprintf("[%s]", conf.ImageStoreName)},
+		NumCPUs:  int32(conf.VCHSize.CPU.Limit),
```

Comment:
This should be conf.ApplianceSize.CPU - the VCH size is for the resource pool that gets created.